### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by laurakwhit**
## What was changed

* ~Updates dependencies~
* Removes `TODOS` with corresponding issues
* ~Fixes TypeScript error~

### CODE REVIEW

**Review by bergundy — APPROVED** [review]

File: webview/src/views/history.svelte [hunk 1]
```diff
@@ -106,6 +106,8 @@
       // case EventType.EVENT_TYPE_WORKFLOW_UPDATE_COMPLETED:
       // case EventType.EVENT_TYPE_WORKFLOW_PROPERTIES_MODIFIED_EXTERNALLY:
       // case EventType.EVENT_TYPE_ACTIVITY_PROPERTIES_MODIFIED_EXTERNALLY:
+      default:
+        return "EVENT"
```
Comment (bergundy) [inline webview/src/views/history.svelte:110]: Please don't do this.

https://github.com/temporalio/vscode-debugger-plugin/pull/26/files#diff-09e34a50f306b6d5d990a30cee8c49edfe22fab10a81955d2d9817d7d735beceR45

Comment (bergundy) [reply webview/src/views/history.svelte:110]: > This function is used to verify any new events are categorized and explicitly does not specify a "default" branch.

Comment (bergundy) [reply webview/src/views/history.svelte:110]: Add the missing event types as needed.

Comment (laurakwhit) [reply webview/src/views/history.svelte:110]: Not having a default here just means we have a Typescript error as the function is expecting a `Category` to be returned. 

And we were functionally setting all `undefined` results to a category [here](https://github.com/temporalio/vscode-debugger-plugin/pull/26/files#diff-09e34a50f306b6d5d990a30cee8c49edfe22fab10a81955d2d9817d7d735beceL126) ⬇️ 
```js 
const category = categorizeEvent(ev.eventType) ?? "EVENT"
```

> This function is used to verify any new events are categorized and explicitly does not specify a "default" branch.

Therefore, I'm not sure how the function verifies anything. Please let me if I'm missing something, but the code still works the same way as before and there was no meaningful message to alert us.

What we could do is throw an error if the event has not been categorized. Would that align with what you're looking for?
```js
   default:
        throw new Error("Event type is not categorized")
```

Comment (bergundy) [reply webview/src/views/history.svelte:110]: It verifies it in compile time

Comment (bergundy) [reply webview/src/views/history.svelte:110]: I don't want the runtime error, I just want compile time safety for when we add more history events

Comment (laurakwhit) [reply webview/src/views/history.svelte:110]: Ah ok, this is not a pattern I'm familiar with. I can remove my changes. 

Another solution here would be to define `EventType` and make it so all new events have to be added to that type (similar to [what we do in the UI](https://github.com/temporalio/ui/blob/main/src/lib/utilities/is-event-type.ts)).

Comment (bergundy) [reply webview/src/views/history.svelte]: That seems like a bad approach, you're redefining all of the protobuf types and there's no type checking that all of the events are accounted for.

**Pull request merged**